public/core.d.ts

```diff
@@ -3838,6 +3838,22 @@ declare namespace JMap {
      **/
     function clearFlashingLocations(): void
 
+    /**
+     * **JMap.Map.displayLayerExtent**
+     *
+     * Displays the layer extent of the given layer Id
+     *
+     * The layer extent will disapear shortly after a pan or a zoom
```

Comment:
c'est toujours vrai qu'il faut bouger la carte pour faire disparaître l'extent si y'a pas de zoom ou pan? sinon, juste dire "`The layer extent will disappear after a few seconds`"

`disappear` 2 "p"